extensions/xcode/src/open-url-in-simulator.command.tsx

```diff
@@ -0,0 +1,37 @@
+import { XcodeSimulatorService } from "./services/xcode-simulator.service";
+import { Clipboard, closeMainWindow, showToast, Toast } from "@raycast/api";
+
+interface OpenSimulatorArguments {
+  url?: string;
+}
+
+export default async (props: { arguments: OpenSimulatorArguments }) => {
+  const toast = await showToast({
+    style: Toast.Style.Animated,
+    title: "Opening URL in Simulator...",
+  });
+  try {
+    const url = props.arguments.url ?? (await Clipboard.readText());
+    if (!url) throw Error("Please provide or copy a valid URL.");
+    const result = await XcodeSimulatorService.openUrl(url);
```

Comment:
What happens if no Simulator is booted? Maybe it would be an option to either check for a booted simulator and show a dedicated error to the user that no simulator is booted or redirect the user to the `Manage Simulators` command to boot a simulator.

Additionally, I think it would be great option for a user to also open URL in the Actions Menu of a Simulator in the `Manage Simulators` command.